VWOSdk/Internal/Core/OperandEvaluator.cs

```diff
@@ -0,0 +1,148 @@
+#pragma warning disable 1587
+/**
+ * Copyright 2019 Wingify Software Pvt. Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma warning restore 1587
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace VWOSdk
+{
+    internal class OperandEvaluator
+    {
+        private static string GROUPING_PATTERN = @"^(.+?)\((.*)\)";
+        private static string WILDCARD_PATTERN = @"(^\*|^)(.+?)(\*$|$)";
+
+        internal OperandEvaluator() {}
+        public bool evaluateOperand(Dictionary<string, dynamic> operandData, Dictionary<string, dynamic> customVariables) {
```

Comment:
method names should be PascalCase instead of camelCase

Reply:
aye